bundles/binding/org.openhab.binding.cups/README.md

```diff
@@ -51,32 +51,34 @@ Each item binding should have this format:
 The CUPS Binding can be used to switch on a printer if there are print jobs in the queue and switch it off if the queue is empty.
 
 #### Items
-
-    Number Print_Jobs_Queued  "Unfinished print jobs"  (FF_Office)   { cups="MX-870#NOT_COMPLETED" }
-    Switch Printer            "Printer"
-
+```
+Number Print_Jobs_Queued  "Unfinished print jobs"  (FF_Office)   { cups="MX-870#NOT_COMPLETED" }
```

Comment:
This is a duplicate of the item shown on line 45, so please delete line 45.
Also, please move the example on line 47 down here.